trace/Sampling.md

```diff
@@ -33,3 +37,26 @@ The OpenCensus library samples based on the following rules:
 3. If the span is a child of a local `Span` the sampling decision will be:
    * If a "span-scoped" `Sampler` is provided, use it to determine the sampling decision.
    * Else keep the sampling decision from the parent.
+
+### QPS sampler implementation details
+The problem we are trying to solve is:
+1. Getting QPS based sampling.
+2. Providing real sampling probabilities.
```

Comment:
What does this mean? Do you mean that we can attach to each span the actual sampling probability in use?

Reply:
That was my idea that we may want to do this.